test/routes/generalTestHelper.test.js

```diff
@@ -150,6 +150,9 @@ module.exports = class GeneralTestHelper {
 
       if (!options.excludeHtmlTests) {
         lab.test(`GET ${routePath} page should have the beta banner`, async () => {
+          if (routePath === '/permit-holder/company/check-name') {
+            console.log('x')
+          }
```

Comment:
This looks like some redundant code?  For debugging?

Reply:
Yes, I was debugging here. Good catch thanks.